packages/terra-form-validation/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ ChangeLog
 Unreleased
 ----------
 
+1.39.0 - (July 28, 2020)
+------------------
+### Changed
+* Minor dependency version bump
+
```

Comment:
Would we need to release this if this is the only change? Could hold on releasing?

Reply:
lerna makes these rules, and keeps all the versions of packages within the mono repo on the same version.